test/load/create_perf_branch.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+git stash --include-untracked
+CURRENT_HEAD=$(git rev-parse HEAD)
+
+git merge max/speed-up-search max/search-database-connection resops/productionize-py-services max/optimize-db-indexes max/increase-pgbouncer-conn-limit max/lower-backend-logging max/test-remove-triggers
```

Comment:
We should probably make a more general solution before merging to master

Reply:
Yeah, I can just not commit this file also. Or potentially rename the branches to be `perf/*`